schutzbot/deploy.sh

```diff
@@ -82,9 +81,10 @@ sudo mkdir -p /etc/osbuild-composer
 sudo cp test/image-tests/osbuild-composer.toml \
     /etc/osbuild-composer/
 
-greenprint "Generating SSL certificates"
-sudo make composer-key-pair
-sudo make worker-key-pair
+greenprint "Copying pre-generated SSL certificates"
+sudo cp /usr/share/tests/osbuild-composer/ca/* \
+    /etc/osbuild-composer/
+sudo chown _osbuild-composer /etc/osbuild-composer/composer-*.pem
```

Comment:
Yeah, this was meant to be a minimal change just to package all tests as RPMs. In a follow-up I think it would make a lot of sense to move more logic out of schutzbot and into the tests themselves.